youritservices/mock-server-bundle/1.0/config/packages/mock_server.yaml

```diff
@@ -0,0 +1,6 @@
+framework:
+    http_client:
+
+mock_server:
+    host_url: '%env(MOCK_SERVER_HOST)%'
+    mocks_directory: '%kernel.project_dir%/tests/apiMocks'
```

Comment:
Would this not fail on `prod` and other non dev, test envs. think this should be:

```yaml
when@test: &test
    mock_server:
        host_url: '%env(MOCK_SERVER_HOST)%'
        mocks_directory: '%kernel.project_dir%/tests/apiMocks'

when@dev: *test
```

Reply:
@alexander-schranz Is this normally something that's required if the package is only used in dev and test and not prod?   If that's the case I'll add add the config for that and open a new PR.  The package is only intended for non-prod environment, as I couldn't see a reason you would want to mock remote API calls in a prod deployment.

Reply:
the config is always loaded and if the bundle is only registered in `dev` and `test` it will show you an error. that not find who is responsible for `mock_server`:

Run:

```
bin/console cache:clear --env prod
```

or switch to `Prod` env. 

```bash
In FileLoader.php line 174:

  There is no extension able to load the configuration for "mock_server" (in "/tmp/project/config/packages/mock_server.yaml"). 
```

Reply:
@alexander-schranz Thanks for the above.  I was only testing this in a current project that's in not in prod at the moment, I'll get that added to the config and keep that in mind for next time.